README.md

```
@@ -78,6 +78,46 @@ This is a Model Context Protocol (MCP) server that allows a large language model
+
+**For GitHub Enterprise Server:**
+```bash
+GH_HOST=github.enterprise.com
+GITHUB_TOKEN=your_enterprise_token
+```
+
+The server automatically constructs API endpoints based on `GH_HOST`:
+- REST API: `https://{GH_HOST}/api/v3`
+- GraphQL API: `https://{GH_HOST}/api/graphql`
+
+**Advanced: Custom API URLs**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)